pandas/tests/frame/test_replace.py

```diff
@@ -1,18 +1,32 @@
 from datetime import datetime
 from io import StringIO
 import re
+from typing import Dict
 
 import numpy as np
 import pytest
 
-from pandas.compat import lrange
-
 import pandas as pd
 from pandas import DataFrame, Index, Series, Timestamp, date_range
 from pandas.tests.frame.common import TestData
 from pandas.util.testing import assert_frame_equal, assert_series_equal
 
 
+@pytest.fixture
+def mix_ab() -> Dict[str, list]:
+    return {'a': list(range(4)),
+            'b': list('ab..'),
+            }
+
+
+@pytest.fixture
+def mix_abc() -> Dict[str, list]:
+    return {'a': list(range(4)),
+            'b': list('ab..'),
+            'c': ['a', 'b', np.nan, 'd'],
+            }
+
```

Comment:
Regarding the fixtures above:

These two dict patterns are used a *lot* in this file and because I needed to convert to ``list(range(4))`` in all locations anyway, I just converted to use fixtures.

Reply:
let me have a look - pretty particular on fixture names